packages/bpk-component-navigation-bar/src/BpkNavigationBar.tsx

```diff
@@ -17,7 +17,7 @@
  */
 
 import type { ReactElement, ReactNode } from 'react';
-import { cloneElement } from 'react';
+// import { cloneElement } from 'react';
```

Comment:
Should this just be removed rather than commented out?

Reply:
Oops, I've done it again - will remove 👍